orchestrator/migrations/versions/schema/2023-06-28_a09ac125ea73_add_throttling_to_refresh_subscriptions.sql

```diff
@@ -0,0 +1,25 @@
+CREATE OR REPLACE FUNCTION refresh_subscriptions_search_view()
+    RETURNS TRIGGER
+    LANGUAGE plpgsql
+AS
+$$
+DECLARE
+    should_refresh     bool;
+    current_epoch      int;
+    last_refresh_epoch int;
+    comment_sql        text;
+BEGIN
+    SELECT extract(epoch from now())::int INTO current_epoch;
+    SELECT coalesce(pg_catalog.obj_description('subscriptions_search'::regclass)::int, 0) INTO last_refresh_epoch;
```

Comment:
How does this work?

Reply:
Ok, after reading the psql docs I understand.

Though I wonder if this is the right function to call as the docs say it's deprecated
![image](https://github.com/workfloworchestrator/orchestrator-core/assets/1482100/7ed2d6f8-a91b-48a1-93aa-639c3a495527)

Also, what does this return on the very first invocation (i.e. when there is no comment yet)?

Reply:
`pg_catalog.obj_description('subscriptions_search'::regclass)` reads the comment (metadata string) on the table. It's converted to an int. `coalesce` ensures that, if there is no data, the timestamp `0` will be used.